sites/native-twin-docs/data.ts

```diff
@@ -0,0 +1,191 @@
+export interface TailwindClass {
+  class: string;
+  web: boolean;
+  native: boolean;
+}
+export const tailwindClasses = {
```

Comment:
I think that constants should follow any convention, for example name with Uppercase: `TAILWIND_CLASSES`